charts/egeria-presentation/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: egeria-presentation
+description: A Helm chart for Kubernetes
```

Comment:
Should describe it's purpose more accurately around the UI as this is what users see when doing a repo search

Reply:
Fixed in f08a6e4ae027